.github/workflows/gen_fedora37_tag.yml

```diff
@@ -84,10 +86,7 @@ jobs:
       - name: "Download artifact"
         uses: actions/download-artifact@v3
         with:
-          name: "fedora33"
-      - name: "Checksum"
```

Comment:
Why is the Checksum step removed? Can it be that `gen_fedora37_continuous.yml` was copied from `gen_fedora36_continuous.yml` prior to #2114 ?

Reply:
Good catch. Fixed.